src/SIL.XForge.Scripture/ClientApp/src/material-styles.scss

```diff
@@ -51,33 +51,81 @@ $mat-sf-orange: (
   )
 );
 
+$mat-sf-error: (
+  100: $errorColor,
+  500: $errorColor,
+  700: $errorColor,
+  contrast: (
+    100: white,
+    500: white,
+    700: white
```

Comment:
I'm still learning all this, but what's the point of defining 100 and 700 if we're just using 500?

Reply:
Because it's the minimum that makes it compile. I know that's a pretty abysmal answer, but it's true and it's about all I actually know. The Material mixins don't like it when we include less. Maybe if I understood how they work better I could find a workaround.